tool/src/org/antlr/v4/codegen/model/Recognizer.java

```diff
@@ -63,13 +65,14 @@ public Recognizer(OutputModelFactory factory) {
 
 		ruleNames = g.rules.keySet();
 		rules = g.rules.values();
-		atn = new SerializedATN(factory, g.atn);
```

Comment:
I'm not a fan of flipping all of these IF to `?:`.  I don't think they are is easy to read.

Reply:
you're doing all sorts of code revisions that are not really needed but are costing my time to review.

Reply:
Sorry.

Reply:
not a problem. You are a valuable contributor and a hardcore tech guy!!  I just think maybe we need to synchronize better :) your instincts seem very good but I have to fit that into my limited focus time.